website/docs/docs/dbt-versions/release-notes/07-June-2023/product-docs-jun.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "June 2023 product docs updates"
+description: "June 2023: The Product docs team merged 132 PRs, made various updates to dbt Cloud and Core, such as the Deploy sidebar, Supported platforms page, added a landing page on the References section, added an ADO example to the CI/CD guide, and more"
+sidebar_label: "Update: Product docs changes"
+tags: [June-2023, product-docs]
+date: 2023-07-04
+sidebar_position: 10
+---
+
+Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 17 new community contributors to docs.getdbt.com &mdash; ✨ aaronbini, sjaureguimodo, aranke, eiof, tlochner95, mani-dbt, iamtodor, monilondo, vrfn, raginjason, AndrewRTsao, MitchellBarker, ajaythomas, smitsrr, leoguyaux, GideonShils, michaelmherrera!
```

Comment:
question: wondering, should we @ mention them? with the rss integration and the community channels, the slack notification could be neat 🤷‍♀️

Reply:
omg i initially added the `@` and then stopped. i can add it but not sure if their git names are the same as their slack names right?

Reply:
yeah, that's true